superwall/src/main/java/com/superwall/sdk/models/entitlements/WebEntitlements.kt

```diff
@@ -7,4 +7,6 @@ import kotlinx.serialization.Serializable
 data class WebEntitlements(
     @SerialName("entitlements")
     val entitlements: List<Entitlement>,
```

Comment:
Is this the EntitlementsResponse equiv? If so we only need the customerInfo now, not these entitlements